src/main/java/controllers/kvis/dao/KvisActivationDAO.java

```diff
@@ -3,16 +3,46 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import controllers.KvisDatabase;
 import controllers.Table;
+import controllers.kvis.dto.kvisAPI.KvisActivatedAPIDTO;
 
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.util.ArrayList;
 
 /**
  * @author Alfred Röttger Rydahl
  * @date 22/11/2021
  **/
 public class KvisActivationDAO
 {
+	public static KvisActivatedAPIDTO[] getAllActiveKvisses() throws SQLException, JsonProcessingException
+	{
+		// Query String
+		final String query = String.format("SELECT * FROM %s", Table.ACTIVE_KVIS.TableName);
+		
+		// Execute and parse
+		return KvisDatabase.queryDatabase(
+				query,
+				new IResultSetParser<KvisActivatedAPIDTO[]>()
+				{
+					@Override
+					public KvisActivatedAPIDTO[] parse(ResultSet resultSet) throws SQLException, JsonProcessingException
+					{
+						ArrayList<KvisActivatedAPIDTO> list = new ArrayList<>(2);
```

Comment:
Hvorfor størrelsen 2?

Reply:
Ja, det er fordi jeg tænkte at den skal i hvert fald være større end 1 eller 0, så derfor gjorde det :smile: Altså det kunne godt være det skulle være 5 eller større, men det er jo uanset hvad bare et gæt for at undgå et par allokationer på heap :)